Update upload-dsyms-bitcode-apps.mdx (#20645)
newrelic/docs-website: src/content/docs/mobile-monitoring/new-relic-mobile-ios/configuration/upload-dsyms-bitcode-apps.mdx

3. Select a specific crash from the <DNT>**Crash types**</DNT> list.
4. Click <DNT>**Upload dSYM**</DNT>. You can either drag and drop your dSYMs directly, or select the file from your computer.

<Callout variant="important">

>   The Upload dSYM button will only appear if the "platform" attribute is native. This button will not be shown in the UI for apps on a hybrid platform such as React Native or MAUI.

Reviewer: ```suggestion
  The Upload dSYM button appears only if the `platform` attribute is set to `native`. This button is not displayed in the UI for apps on hybrid platforms such as React Native or MAUI.
```
